<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>PhotoBooth</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="FR_photolive">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>505</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/img/photo_bienvenue.png);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_photo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 6px</string>
    </property>
    <property name="text">
     <string> Prendre 
une photo</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="LCD_timer">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>20</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QTextEdit" name="TB_disp">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_valider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>96</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 6px</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_annuler">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>96</width>
      <height>48</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 6px</string>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_parametre">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>34</width>
      <height>34</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(225, 225, 225);
border-radius: 6px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/img/photo_parametre.png</normaloff>:/img/photo_parametre.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>605</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
